core/src/fees.ts

```diff
@@ -23,6 +26,12 @@ export const getApproximateTransactionFees = async function (network: string): P
     };
 };
 
+export const getApproximateTransactionFees = memoizee(_getApproximateTransactionFees, {
+    maxAge: TRANSACTION_FEE_CACHE,
+    promise: true,
+    length: 1,
+});
```

Comment:
As suggested already: can you please cache actual function that fetches gas price instead (and please use smaller cache, something like 5 seconds)?
As `getExchangeRateBySymbol` is already cached, as far as I remember.

Reply:
> * As suggested [above](https://github.com/sidestream-tech/unified-auctions-ui/pull/82#issuecomment-1028081957): can you please cache actual function that fetches gas price instead (and please use smaller cache, something like 10 seconds)?

- [X] Cached the Gas price for 10 seconds
- [X] Removed cache on parent function